test/Microsoft.AspNetCore.Razor.Design.Test/IntegrationTests/PackIntegrationTest.cs

```diff
@@ -15,6 +15,46 @@ public PackIntegrationTest(BuildServerTestFixture buildServer)
         {
         }
 
+        [Fact]
+        [InitializeTestProject("ClassLibrary")]
+        public async Task Pack__NoBuild_Works_IncludesRazorAssembly()
```

Comment:
nit: two underscores after `Pack`

Reply:
Could you also add a test that verifies we do add the files in no-build scenarios if the flag is set?